crates/trigger/src/runtime_config.rs

```diff
@@ -57,12 +57,12 @@ impl RuntimeConfig {
         Ok(())
     }
 
-    /// Return a Vec of configured [`spin_config::Provider`]s.
-    pub fn config_providers(&self) -> Vec<ConfigProvider> {
-        let default_provider = ConfigProviderOpts::default_provider_opts(self).build_provider();
-        let mut providers: Vec<ConfigProvider> = vec![default_provider];
+    /// Return a Vec of configured [`spin_variables::Provider`]s.
```

Comment:
This was existing code but it's a bit odd that the doc string says the Vec is of `spin_variables::Provider` and the very next line says it's of `VariablesProvider`!

Reply:
Updated to `VariablesProvider`.